frameworks/person-escort-record/README.md

```diff
@@ -76,10 +76,15 @@ Question files should live in [`questions/`](./questions) and be stored in a fla
 
 ### Question options
 
-- `type` **(required)** - the type of question input. Current supported types: `radio`, `checkbox`, `text`, `textarea`
+- `type` **(required)** - the [type of question](#question-types) input. Current supported types: `radio`, `checkbox`, `text`, `textarea`, `add_multiple`
```

Comment:
I wonder if we drop the second sentence and describe the types below to avoid needing to duplicate them?